src/App.js

```diff
@@ -1,29 +0,0 @@
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
-
-export default App;
```

Comment:
- [ ] You shouldn't delete the App component make sure to add it to the index component and it should be classed based not function then put the `Calculator.js` component in the App.js in order to match the requirements 
-Screenshot 
<img width="767" alt="Screen Shot 2022-10-26 at 15 50 55" src="https://user-images.githubusercontent.com/61790587/198059486-997a655f-558d-464e-815b-899cecdf1e62.png">

Reply:
hi @youmari , 
I have already refactored the project, 
I don't think I have this App.js in the new branch "components"
![Screenshot 2022-10-26 150433](https://user-images.githubusercontent.com/38554511/198063877-82eb409b-cf38-4e3d-a4a8-df22e2caae96.png)